firebase.json

```diff
@@ -1,9 +1,6 @@
 {
   "functions": {
-    "predeploy": [
-      "npm --prefix \"$RESOURCE_DIR\" run lint",
```

Comment:
Why remove the linting stage? Will type errors still be caught on build?

Reply:
Added it back, it was during my debug checking process.